src/main/java/com/pinterest/secor/common/SecorConfig.java

```diff
@@ -397,4 +411,19 @@ private long getLong(String name) {
     private String[] getStringArray(String name) {
         return mProperties.getStringArray(name);
     }
+
+    // This method can be useful if you use include and have several times the same key
+    // Retrieve the last children file value
+    @VisibleForTesting
+    String getStringLastValue(String name, String defaultValue) {
```

Comment:
Do you really this method?  This behavior is different than all the other property value which we got from the first occurrence.